sickbeard/providers/newznab.py

```diff
@@ -141,9 +141,11 @@ def _get_season_search_strings(self, ep_obj):
 
         # search
         rid = helpers.mapIndexersToShow(ep_obj.show)[2]
+        #added exception for usenet-crawler, as it seems they have changed something within their api
+        tvrage = 'rid' if self.name <> "Usenet-Crawler" else 'tvrage' 
```

Comment:
I'm just doing to many different languages at the moment. <> works the same as !=. <> is supported, but i just read it's deprecated. If you'd like I can change it to !=.

Personally i only use "not in", with lists or tuples. But also that's possible, if you prefer, I can also use that.